fdbserver/LatencyBandsMap.actor.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * LatencyBandsMap.actor.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "fdbserver/LatencyBandsMap.h"
+#include "flow/UnitTest.h"
+#include "flow/actorcompiler.h" // must be last include
+
+LatencyBands* LatencyBandsMap::getLatencyBands(TransactionTag tag) {
```

Comment:
Using `Optional` rather than `nullptr` might better communicate the behavior of this function, though it's probably less of a concern given that it's not public